File: Project-Euler/Stats/euler15/euler15.py
import sys

gridSize = int(sys.argv[1])

grid = [ [ 1 for i in xrange(gridSize+1) ] for j in xrange(gridSize+1) ]

def calc(grid, gridSize):
  for i in xrange(gridSize-1, -1, -1):
    for j in xrange(gridSize-1, -1, -1):
      grid[i][j] = grid[i+1][j] + grid[i][j+1]
  return grid[0][0]

print calc(grid, gridSize)

File: Project-Euler/Stats/euler15/euler15.cpp
//
//  main.cpp
//  Euler15
//
//  Created by Sonni on 20/05/16.
//  Copyright © 2016 Sonni Jensen. All rights reserved.
//

#include <iostream>

unsigned long calc(unsigned int gridSize)
{
    unsigned long grid[gridSize + 1][gridSize + 1];
    for (unsigned int i = 0; i < gridSize+1; i++)
        for (unsigned int j = 0; j < gridSize+1; j++)
            grid[i][j] = 1;
    
    for (int i = gridSize - 1; i >= 0; i--)
        for (int j = gridSize - 1; j >= 0; j--)
            grid[i][j] = grid[i+1][j] + grid[i][j+1];
        
    
    return grid[0][0];
}

int main(int argc, const char * argv[]) {
    std::cout << calc(atoi(argv[1])) << std::endl;;
    return 0;
}
